specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/examples/GetAllTestFiles.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "api-version": "2022-06-01-preview"
```

Comment:
Does it not take any parameter

Reply:
Seems like testId params has not been included by oav tool while generating examples. I have added those back